app/components/TracksCard/index.js

```diff
@@ -0,0 +1,46 @@
+/**
+ *
+ * TracksCard
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Card, Image } from 'antd';
+import If from '@components/If';
+import { isEmpty } from 'lodash';
```

Comment:
put all the library imports to the top, followed by component imports. So `lodash` should go top of If component import